datasets/titanic/metadata.json

```diff
@@ -1,11 +1,23 @@
 {
     "@context": {
+      "@vocab": "https://schema.org/",
       "sc": "https://schema.org/",
       "ml": "http://mlcommons.org/schema/",
-      "wd": "https://www.wikidata.org/wiki/Q"
+      "wd": "https://www.wikidata.org/wiki/Q",
+      "dataType": {
+        "@id": "ml:dataType",
+        "@type": "@id"
+      },
+      "field": {
+        "@id": "ml:Field",
+        "@type": "@id"
+      },
+      "recordSet": {
+        "@id": "ml:RecordSet",
+        "@type": "@id"
+      }
     },
-
-    "@type": "sc:Dataset",
+    "@type": "Dataset",
```

Comment:
Is there a reason to drop the "sc:" prefix?
This was done to make it more obvious to spot what is coming from where (here schema.org).
If we can keep the prefix, I think it makes the config more readable.

Reply:
Done.